drivers/mipi_dbi/mipi_dbi_smartbond.c

```diff
@@ -64,6 +64,10 @@ struct mipi_dbi_smartbond_data {
 	volatile bool underflow_flag;
 	/* Layer settings */
 	lcdc_smartbond_layer_cfg layer;
+#if defined(CONFIG_PM_DEVICE) || defined(CONFIG_PM_DEVICE_RUNTIME)
```

Comment:
Just a suggestion- if you group this `bool` declaration with the `bool` above, it should save flash space as both values can be packed into a 4 byte alignment

Reply:
I guess that grouping should be achieved via bit field structs, right?

Reply:
It can be- even if you just declare the `bool` values next to each other in the struct you'll gain a benefit. The structure's minimum alignment should be given by the largest field in the structure (which is probably the `lcdc_smartbond_layer_cfg` structure. If you place the `bool` declaration with the other `bool` declaration, both declarations can fit into one block of that minimum alignment size. Otherwise, the compiler is likely placing padding after the first `bool` declaration up to the alignment of the `lcdc_smartbond_layer_cfg` structure, then placing the next `bool` declaration after that.

For more info: http://www.catb.org/esr/structure-packing/

Again, only a suggestion, and not something you need to worry too much about implementing- just always nice to save a bit of space

Reply:
Nice catch :)